packages/message-list/src/vaadin-message-list-mixin.d.ts

```diff
@@ -39,4 +39,9 @@ export declare class MessageListMixinClass {
    * ```
    */
   items: MessageListItem[] | null | undefined;
+
+  /**
+   * When set to `true`, the message text is parsed as Markdown.
+   */
+  markdown: boolean | undefined;
```

Comment:
Suggestion: Instead of markdown boolean - what would you think about a property / enum `mode` allowing to switch between mode=text (default), mode=`markdown` or `html` or any other thing you guys might come up with?

Reply:
This option was discussed during API design. We ended up with a `markdown` flag due to simplicity and because markdown also allows `html` content.

Reply:
We also discussed later adding some kind of a `renderer` API which would allow you to decide how an item text gets rendered inside the message. That's not included in the current scope though.